src/write/range.rs

```diff
@@ -54,16 +54,16 @@ impl RangeListTable {
         version: u16,
         address_size: u8,
     ) -> Result<RangeListOffsets> {
-        if version < 5 {
-            Ok(RangeListOffsets {
+        match version {
+            3...4 => Ok(RangeListOffsets {
```

Comment:
Both gcc and llvm write `.debug_ranges` even for DWARF version 2, so we should do the same.